File: backend/app/api/routes/health.py
from fastapi import APIRouter, Request

router = APIRouter()


@router.get("/health")
def read_health(request: Request) -> dict[str, str]:
    settings = request.app.state.settings
    return {
        "status": "ok",
        "service": settings.service_name,
        "version": settings.version,
        "data_dir": settings.data_dir.as_posix(),
    }

File: apps/web/src/lib/api/health.ts
import type { ApiRequestClient } from './client'

export type HealthResponse = {
  data_dir: string
  service: string
  status: string
  version: string
}

export function createHealthApi(client: ApiRequestClient) {
  return {
    getHealth: () => client.get<HealthResponse>('/health'),
  }
}
